parent/pom.xml

```diff
@@ -952,6 +953,11 @@
                 <artifactId>camel-chunk</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel</groupId>
+                <artifactId>camel-cifs</artifactId>
```

Comment:
Is thius really needed?

Reply:
Man, that's not needed. That's very strange. That was the old name I started this component with. I'll erase my local cache and rebuild. Thanks for pointing it out!